docs/src/bidding_formats.md

```diff
@@ -17,24 +17,239 @@ Note that, in case the heuristic bid alternative is chosen, bid files will be ge
+
+In the following example of a `parent_profile.csv` file, we have have bids for the Bidding Groups `bg_1` and `bg_2`, with three profiles each. 
+For `bg_2`, its first profile is a predecessor of its second profile, while for `bg_1`, its second profile is a predecessor of its third profile. 
+
+
+
+| period | profile |  bg_1  |  bg_2  |
+|:------:|:-------:|:------:|:------:|
+|   1    |    1    |  0.0   |  0.0   |
+|   1    |    2    |  0.0   |  1.0   |
+|   1    |    3    |  2.0   |  0.0   |
+|   2    |    1    |  0.0   |  0.0   |
+|   2    |    2    |  0.0   |  1.0   |
+|   2    |    3    |  2.0   |  0.0   |
+
+
+
+
+### Complementarity constraints
+
+Complementarity constraints impose that the sum of the activation coefficients of a complementary group must be less than or equal to 1. 
+This is represented in the database by the `complementary_grouping_profile` time series file, which can be attached to the case using the `[IARA.link_time_series_to_file](@ref)` function.
+
+
+In the following example of a `complementary_grouping_profile.csv` file, we have two bidding groups, `bg_1` and `bg_2`, with three profiles each.
+The columns `bg_1` and `bg_2` contain boolean values that indicate whether the profile is part of the complementary group or not.
+
+Observing the table, we can draw the following conclusions:
+- The first profile of `bg_2` is part of the complementary group 1, along with its second profile.
+- The first profile of  `bg_2` is also part of the complementary group 2, but no other profile is part of this group.
+- The second profile of `bg_2` is part of the complementary group 3, with no other profile being part of this group.
+- No profile of `bg_1` is part of any complementary group.
+
+If a profile is not part of any complementary group, its coefficient keeps its original constraint of being between 0 and 1.
+The same applies for profiles that are the only ones in a complementary group.
+
+
+
+
+| period | profile | complementary_group |  bg_1  |  bg_2  |
+|:------:|:-------:|:-------------------:|:------:|:------:|
+|   1    |    1    |          1          |  0.0   |  1.0   |
+|   1    |    1    |          2          |  0.0   |  1.0   |
+|   1    |    1    |          3          |  0.0   |  0.0   |
+|   1    |    2    |          1          |  0.0   |  1.0   |
+|   1    |    2    |          2          |  0.0   |  0.0   |
+|   1    |    2    |          3          |  0.0   |  1.0   |
+
+
+
+
+### Minimum activation constraints
+
+Minimum activation constraints impose that, if a profile bid is accepted, then its coefficient must be greater than or equal to a certain threshold.
```

Comment:
Minimum activation constraints establish that, for a profile bid to be accepted, the offered quantity must be at least a certain threshold, expressed as a percentage of the total bid volume.